requirements.txt

```diff
@@ -5,7 +5,7 @@ fiona
 gcsfs
 geopandas>=0.8
 google_apitools
-google-cloud-aiplatform>=1.35.0
+google-cloud-aiplatform>=1.48.0
```

Comment:
@rsadiq Hi Rizwan, can you explain why this library version needs to be bumped?

Reply:
Hi @jzxu , it was because of installation issue of the specific version and its compatibility with the other packages.